Frontend/android-native/fastlane/Fastfile

```
@@ -22,12 +22,31 @@
+
+
 platform :android do
   desc "Runs all the tests"
   lane :test do
     gradle(task: "test")
   end
 
+  desc "My awesome app"
+  lane :firebase_distribute do
+
+    gradle(
```

RuboCop (1.73)

[convention] Extra empty line detected at block body beginning.

(Layout/EmptyLinesAroundBlockBody)